packages/calypso-color-schemes/bin/build-css.js

```diff
@@ -2,13 +2,31 @@
 const { dirname, join } = require( 'path' );
 const { existsSync, mkdirSync, writeFileSync } = require( 'fs' );
 const { renderSync } = require( 'node-sass' );
+const postcss = require( 'postcss' );
+const postcssCustomProperties = require( 'postcss-custom-properties' );
 
 const INPUT_FILE = join( __dirname, '..', 'src', 'calypso-color-schemes.scss' );
-const OUTPUT_FILE = join( __dirname, '..', 'dist', 'calypso-color-schemes.css' );
+const OUTPUT_FILE = join( __dirname, '..', 'css', 'index.css' );
+const OUTPUT_JS_FILE = join( __dirname, '..', 'js', 'index.js' );
 
 if ( ! existsSync( dirname( OUTPUT_FILE ) ) ) {
 	mkdirSync( dirname( OUTPUT_FILE ), { recursive: true } );
 }
 
+if ( ! existsSync( dirname( OUTPUT_JS_FILE ) ) ) {
+	mkdirSync( dirname( OUTPUT_JS_FILE ), { recursive: true } );
+}
+
 const output = renderSync( { file: INPUT_FILE } );
+
 writeFileSync( OUTPUT_FILE, output.css );
+
+// export CSS properties into a JavaScript file, so whoever uses it doesn't need to parse it over and over
+( async function () {
+	await postcss( [
```

Comment:
Top-level await in Node.js will make this much nicer once it ships 🙂

If the `postcss` call fails, i.e., returns a rejected `Promise`, Node.js will report it as unhandled rejection. That could be improved.